modular_skyrat/modules/colony_fabricator/code/design_datums/tools.dm

```diff
@@ -31,7 +32,19 @@
 	)
 
 // Crowbar that is completely normal except it can force doors
+/datum/design/colony_door_crowbar_replacement
+	name = "Crowbar"
+	id = "colony_crowbar"
+	build_type = COLONY_FABRICATOR
+	build_path = /obj/item/crowbar/red
+	materials = list(
+		/datum/material/iron = SHEET_MATERIAL_AMOUNT * 1.75,
+	)
+	category = list(
+		RND_CATEGORY_INITIAL,
+	)
```

Comment:
This confuses me. Is it the replacement prybar that's supposed to break?

Reply:
I forgot to adjust the material price, currently, that's a standard crowbar that costs "*0.5" not "*1.75" and it'll be until I get around to making the prybar break (which may not be until next weekend.)